vaults-monitor/api/src/btc-vaults.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const { esploraClient } = require('esplora-client')
+const {
+  getBitcoinChainLastHeader,
+  getBitcoinCustodyAddress,
+  getBitcoinVaultStateAddress,
+  getPendingWithdrawalAmountSat,
+  getPendingWithdrawalCount,
+  getTunnelManagerStatus,
+  getVaultByIndex,
+  getVaultCounter,
+  getVaultStatus,
+  hemi,
+  hemiSepolia,
+} = require('hemi-viem')
+const viem = require('viem')
+
+function getHemiClient(chainId) {
+  const chain = { [hemi.id]: hemi, [hemiSepolia.id]: hemiSepolia }[chainId]
+  const transport = viem.http(chain.rpcUrls.default.http[0])
+  return viem.createPublicClient({ chain, transport })
```

Comment:
This is the default value (See [here](https://github.com/wevm/viem/blob/4ccc559d243796387c671f69694f475fc5edc312/src/clients/transports/http.ts#L110)) so we can omit it

```suggestion
  const transport = viem.http()
  return viem.createPublicClient({ chain, transport })
```